pkgs/by-name/dr/druid/package.nix

```
@@ -0,0 +1,69 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, extensions ? { }
+, libJars ? [ ]
+, nixosTests
+, mysqlSupport ? true
+}:
+let
+  inherit (lib) concatStringsSep mapAttrsToList optionalString forEach;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apache-druid";
+  version = "29.0.0";
+  src = fetchurl {
+    url = "https://dlcdn.apache.org/druid/${finalAttrs.version}/${finalAttrs.pname}-${finalAttrs.version}-bin.tar.gz";
+    sha256 = "sha256-Ka/coMWYXDjP18hgwwJq+xzSO7tL7/XtIZRcekpX6g4=";
+  };
+  mysqlConnector = fetchurl {
+    url = "https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.48/mysql-connector-java-5.1.48.jar";
+    hash = "sha256:1a7cnr4r03mv7j0d6sji07m892ygcr7wgya4mzj5l7w2lfm6rqjn";
+  };
+  dontBuild = true;
+  passthru = {
+    tests = nixosTests.druid.default.passthru.override {
+      druidPackage = finalAttrs.finalPackage;
+    };
+  };
```

```suggestion
  passthru.tests = nixosTests.druid.default.passthru.override {
    druidPackage = finalAttrs.finalPackage;
  };
```
[optional] remove unnecessary nesting